external/llvm-project/mlir/lib/Conversion/TosaToLinalg/TosaToLinalgNamed.cpp

```diff
@@ -1130,7 +1130,7 @@ void mlir::tosa::populateTosaToLinalgNamedConversionPatterns(
   }
   patterns->add<
       // clang-format off
-      ConvConverter<tosa::Conv2DOp, linalg::Conv2DNhwgcGfhwcOp, linalg::Conv2DNhwgcGfhwcQOp>,
+      ConvConverter<tosa::Conv2DOp, linalg::Conv2DNhwgcGfhwcOp, linalg::Conv2DNgchwGfchwQOp>,
```

Comment:
Wait hold on I think this is a bug?

Reply:
You are right, I noticed it and fixed it in the next commit eaa86317f917052c6e9ab14742871e8189750182